guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

```diff
@@ -2734,10 +2736,10 @@ WEBVTT
 [Stephen Frost]: Does that mean that they're disorganized or...?
 
 01:41:31.588 --> 01:41:34.031
-[Stephen Frost]: They're unbalanced.
+[]: Unbalanced.
```

Comment:
We don't know who said this? Should we use a [?] or just leave it [ ]?

Reply:
I checked again and it's Dave Rasnick who says "They're unbalanced."  Please change Line 2739 to read [Dave Rasnick]: They're unbalanced.

He says it twice (see Line 2742 below).